coderd/templates_test.go

```diff
@@ -1618,6 +1618,69 @@ func TestPatchTemplateMeta(t *testing.T) {
 		require.NoError(t, err)
 		assert.False(t, updated.UseClassicParameterFlow, "expected false")
 	})
+
+	t.Run("SupportEmptyOrDefault", func(t *testing.T) {
+		t.Parallel()
+
+		client := coderdtest.New(t, nil)
+		user := coderdtest.CreateFirstUser(t, client)
+		version := coderdtest.CreateTemplateVersion(t, client, user.OrganizationID, nil)
+
+		description := "test-description"
+		icon := "/icon/icon.png"
+		defaultTTLMillis := 10 * time.Hour.Milliseconds()
+
+		template := coderdtest.CreateTemplate(t, client, user.OrganizationID, version.ID, func(ctr *codersdk.CreateTemplateRequest) {
```

Comment:
A bit overkill, but since `template.Description` is the reference for the test cases.

```suggestion
		template := coderdtest.CreateTemplate(t, client, user.OrganizationID, version.ID, func(ctr *codersdk.CreateTemplateRequest) {
		require.Equal(t, description, template.Description)
		require.Equal(t, icon, template.Icon)
```